qa/SidechainTestFramework/scutil.py

```diff
@@ -470,6 +483,7 @@ def start_sc_node(i, dirname, extra_args=None, rpchost=None, timewait=None, bina
     url = "http://rt:rt@%s:%d" % ('127.0.0.1' or rpchost, sc_rpc_port(i))
     proxy = SidechainAuthServiceProxy(url)
     proxy.url = url  # store URL on proxy for info
+    proxy.cfgFileName = cfgFileName # store the name of the cfg file with path
```

Comment:
instead of storing `cfgFileName`, why not to store `datadir`? Then we may use `datadir` to access the storages of given node.

Reply:
That is because actual dataDir is specified in the configuration file, for instance:

`sc_settings.conf:    dataDir = /tmp/scorex/data/blockchain`

But ok, since in the test framework we use:

`/qa/resources/template.conf:  dataDir = "%(DIRECTORY)s/sc_node%(NODE_NUMBER)s/blockchain"`

we might as well use `datadir + / + "blockchain"`